### ISSUE TITLE
Feature Request: ability to declare help file opened by right click>help

### ISSUE THREAD

**Issue by ulioidle** [pure-data/pure-data#1388]
This would simplify documentation and allow making it more intuitive for the user, like objects can be grouped into a single help file like text/array/list are and fewer documents to maintain and keep track of.

**Comment by Spacechild1** [comment]
This already exists for writing externals (`class_sethelpsymbol`). Are you maybe thinking about adding this functionality for abstractions? I see how this could be useful.

Could be something simple like `[help <helpsymbol>]`. It could store the help symbol somewhere in the canvas structure (probably in `gl_privatedata`), so that it can be queried in `canvas_done_popup`.

**Comment by ulioidle** [comment]
Yes, abstractions. Seems to me that it would fit in with [declare], no need for a new object, it is just declaring a file after all.

**Comment by Spacechild1** [comment]
yes, `[declare -help <name>]` would also be a possibility.
